docs/src/pages/demos/text-fields/TextFields.tsx

```diff
@@ -52,7 +52,7 @@ interface State {
 
 function TextFields() {
   const classes = useStyles();
-  const [values, setValues] = React.useState({
+  const [values, setValues] = React.useState<State>({
```

Comment:
I believe the state is implicit. Should we repeat it?

Reply:
We don't have to, but without it `values` is not of type `State` it's just an object that looks like `State`